app/Http/Controllers/HomeController.php

```diff
@@ -49,4 +47,10 @@ public function store(Request $request){
         $store->save();
         return Redirect::to('home')->with('success', 'successfully submited');
     }
+
+    public function show()
+    {
+        $data = Cases::get();
```

Comment:
Pagination must be used, if you a million cases your server will be crashed because you get it all from databases.

Reply:
yes